src/SFE.scss

```diff
@@ -34,7 +34,6 @@
   font-weight: $font-weight-base;
   line-height: $line-height-base;
   color: $body-color;
-  text-align: left;
```

Comment:
Just curious, what was this style breaking?

Now that I am thinking about this style I think it might break RTL support, so it's probably better that it is left out.